client/signup/wpcom-login-form/index.jsx

```diff
@@ -33,6 +39,7 @@ export default function WpcomLoginForm( { redirectTo, authorization, pwd = '', l
 			<input type="hidden" name="pwd" value={ pwd } />
 			<input type="hidden" name="authorization" value={ authorization } />
 			<input type="hidden" name="redirect_to" value={ redirectTo } />
+			{ rememberMe ? <input type="hidden" name="rememberme" value="true" /> : undefined }
```

Comment:
Does it make sense to keep the value as "forever" like in WP core?

https://github.com/WordPress/wordpress-develop/blob/8711aa5ab60e35d78dab73e316674b59b90246da/src/wp-login.php#L1537

```suggestion
			{ rememberMe ? <input type="hidden" name="rememberme" value="forever" /> : undefined }
```

Reply:
Makes sense, I've now changed it to `forever`. The behaviour is the same with `true` or `forever`.

Reply:
Yep - it doesn't matter which string we convert to boolean; it will always be `true` in the end. This nit was aimed mostly at precision/consistency.

Reply:
I understand, I agree with your suggestion of using `forever`, I was just mentioning that the behaviour doesn't change to inform readers of the PR that it still functions as it did when the value was `true` :)